examples/Makefile

```diff
@@ -122,6 +138,7 @@ IMAGE_TP_US ?= quay.io/bpfd-userspace/go-tracepoint-counter:latest
 IMAGE_XDP_BC ?= quay.io/bpfd-bytecode/go-xdp-counter:latest
 IMAGE_XDP_US ?= quay.io/bpfd-userspace/go-xdp-counter:latest
 KUST_DIR=default
+KIND_CLUSTER_NAME ?= bpfd-deployment
```

Comment:
Add `KIND_CLUSTER_NAME` to the Makefile comments section above, something like:
```
KIND_CLUSTER_NAME: ## Name of the deployed cluster to load to, defaults to `bpfd-deployment`
```

Reply:
Done